R/mod_metricGrid.R

```diff
@@ -50,7 +50,9 @@ metricGridServer <- function(id, metrics) {
         metricBoxServer(id = metric['name'],
                         title = metric['title'],
                         desc = metric['desc'],
-                        value = metric['value'],
+                        value = dplyr::case_when(metric['name'] != 'has_bug_reports_url' ~ metric['value'],
+                                                 metric['value'] == "1" ~ 'TRUE',
+                                                 TRUE ~ 'FALSE'),
```

Comment:
FYI @Jeff-Thompson12, this needs to be implemented in each report as well, since the change was implemented at the metricBox-level (which is appropriate) and each report Rmd file defines a function on how to build the boxes. I'm adding this to #414

Reply:
Okay, I will also add a safety valve in for the URL argument being passed as well if not already done.